frontend/src/assets/translations/nl.json

```diff
@@ -0,0 +1,30 @@
+{
+  "about": "Over ons",
+  "volunteers": "Vrijwilligers",
+  "contactus": "Contacteer Ons",
+  "signup": "Aanmelden",
+  "login": "Inloggen",
+  "findyourvolunteer": "VIND JE BENEDENDE",
+  "volunteerswhocanhelp": "VRIJWILLIGERS WELKE HET KAN HELPEN",
+  "skill": "Vaardigheid",
+  "city": "Stad",
+  "language": "Taal",
+  "whoarewe": "WIE ZIJN WE?",
+  "aboutbehelp1": "BeHelp is een dienst die reisden en asielverzoekers verbindt met vrijwilligers die zich kunnen delen met hun ervaring en vaardigheden pro bono.",
+  "aboutbehelp2": "Op BeHelp kunt u vrijwilligers vinden van alle landen.",
+  "ourservices": "Onze diensten",
+  "ourservices2": "BeHelp is een groep vrijwilligers die tijd, willen en mogelijkheid hebben om nieuwgeborenen in België te helpen met dagelijks problemen.",
```

Comment:
1)Behelp is een groep van vrijwilligers die tijd  geeft aan nieuwe belgische inkomers met hun dagelijkse problemen.
2)Behelp is een groep van vrijwilligers die de mogelijkheid, tijd en wensen hebben om aan nieuwkomers met hun dagelijkse problemen te helpen.